pkg/apiserver/resthandler.go

```diff
@@ -500,7 +500,16 @@ func PatchResource(r rest.Patcher, scope RequestScope, typer runtime.ObjectTyper
 			return
 		}
 
-		result, err := patchResource(ctx, timeout, versionedObj, r, name, patchType, patchJS, scope.Namer, scope.Codec)
+		updateAdmit := func(updatedObject runtime.Object) error {
+			if admit != nil && admit.Handles(admission.Update) {
+				userInfo, _ := api.UserFrom(ctx)
+				return admit.Admit(admission.NewAttributesRecord(updatedObject, scope.Kind.GroupKind(), namespace, name, scope.Resource.GroupResource(), scope.Subresource, admission.Update, userInfo))
```

Comment:
> Why does update not take both old and new objects?

That was more than I wanted to tear off here.  This pull closes up an existing gap in admission logic.  Restructuring admission of updates is beyond what I wanted to do to close off the field level admission failures.